Transbank/Transbank.csproj

```
@@ -16,7 +16,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <PackageTags>payments, cl, chile, transbank</PackageTags>
     <PackageIconUrl>https://www.transbankdevelopers.cl/favicon.ico</PackageIconUrl>
-    <VersionPrefix>2.6.1</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>

```

Should it be version 3.0.0?
```suggestion
    <VersionPrefix>3.0.0</VersionPrefix>

```

---

It's fine, you can check it in the last part of the readme

> Modificar Transbank.csproj para que <VersionPrefix> sea X.Y.{Z+1} (de manera que los pre-releases que se generen después del release sean de la siguiente versión).